_posts/0000-01-03-gyenney.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![panda-birthday](https://cloud.githubusercontent.com/assets/16547949/25400994/60f64804-29c2-11e7-8a6d-4401d7d294b9.jpg)
 {: .center}
 
-Caption here
+Don't poke the bear, especially when he's eating his birthday cake!
```

Comment:
Good caption!!

Reply:
You may want to add markdown to your caption
`Don't poke the bear, especially when he's eating his birthday cake!`

Reply:
I have now updated the caption with markdown.

Reply:
How is it now?